automatic/ketarin/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,13 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsPath = Split-Path -Parent $MyInvocation.MyCommand.Definition
+
+$packageArgs = @{
+  packageName = $env:ChocolateyPackageName
+  file        = "$toolsPath\"
+  destination = $toolsPath
+}
+
+Get-ChocolateyUnzip @packageArgs
+Set-Content -Path "$toolsPath\Ketarin.exe.gui" -Value ""
```

Comment:
This isn't required any more AFAIK.

Reply:
No harm to have it though, so a gui shim is sure to be made both for new and older versions of chocolatey.